### DISCUSSION TITLE
无法获取general/无法获取DNS部分的……

### DISCUSSION THREAD

**Discussion by dumplingmanG** [vernesong/OpenClash#3260] [General]
错误: 无法获取General部分的转发端口设置, OpenClash 使用原始配置文件启动失败
错误: 无法获取DNS部分的运行模式设置, OpenClash 使用原始配置文件启动失败
错误: 无法获取DNS部分的监听端口设置, OpenClash 使用原始配置文件启动失败
错误: OpenClash 启动失败，尝试使用原始配置文件启动...
第六步: 等待主程序下载外部文件...
第五步: 检查内核启动状态...

自动更新订阅经常会出现以上内容，必须要手动更新订阅才恢复正常。有什么办法可以让openclash的自动更新订阅不出现以上错误。

Openclash DNS设置 ： 本地劫持 使用dnsmasp转发 勾选了禁止缓存DNS 。其他覆写部分均无任何设置 无勾选  

防火墙自定义规则为 

iptables -t nat -A PREROUTING -p udp --dport 53 -j REDIRECT --to-ports 53
iptables -t nat -A PREROUTING -p tcp --dport 53 -j REDIRECT --to-ports 53
[ -n "$(command -v ip6tables)" ] && ip6tables -t nat -A PREROUTING -p udp --dport 53 -j REDIRECT --to-ports 53
[ -n "$(command -v ip6tables)" ] && ip6tables -t nat -A PREROUTING -p tcp --dport 53 -j REDIRECT --to-ports 53
![微信图片_20230504220530](https://user-images.githubusercontent.com/132582884/236231986-0c0af7db-e19d-4f34-ab59-5ff45101d5c2.png)
![微信图片_20230504220452](https://user-images.githubusercontent.com/132582884/236232005-033a4c9f-341a-4a7b-a9cf-a204d5a40644.png)
![微信图片_20230504220414](https://user-images.githubusercontent.com/132582884/236232019-8170aca0-2c79-48fe-8d9b-7899a93d6a6f.png)

**Comment by wuyuqq** [comment]
遇到同样的问题

**Reply by Lizekai1121** [reply]
同
